lxc/launch.go

```diff
@@ -33,7 +33,11 @@ lxc launch ubuntu:22.04 u2 -t aws:t2.micro
     Create and start a container using the same size as an AWS t2.micro (1 vCPU, 1GiB of RAM)
 
 lxc launch ubuntu:22.04 v1 --vm -c limits.cpu=4 -c limits.memory=4GiB
-    Create and start a virtual machine with 4 vCPUs and 4GiB of RAM`))
+    Create and start a virtual machine with 4 vCPUs and 4GiB of RAM
+
+lxc launch ubuntu:22.04 v1 --vm -c limits.cpu=2 -c limits.memory=8GiB -d root,size=32GiB
+    Create and start a virtual machine with 4 cpus, 8GiB of RAM and a root disk of 32GiB`))
```

Comment:
2 CPUs

Reply:
Hmm, thanks for catching my copy-n-paste error on the count. I now notice we're not very consistent with `cpus`, `vCPUs` and `CPUs`. I'll make them all the same, let's agree on which variant, `vCPUs`?

Reply:
Yeah that sounds fine